microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -131,6 +131,21 @@ def _enable_hugepages(node: Node) -> None:
     )
 
 
+def _force_dpdk_stable_lkg_for_ubuntu_18_04(
```

Comment:
The method may be extended to other distros. So change the name like `_set_source_by_distro`